feconf.py

```diff
@@ -469,6 +469,9 @@ def get_empty_ratings():
 # A sorted list of default categories.
 DEFAULT_CATEGORIES = sorted(CATEGORIES_TO_COLORS.keys())
 
+#Size of the gravatar in px assigned to the user after signup
+GRAVATAR_SIZE_PX = 50
```

Comment:
Not a good reason. Can you perhaps match the size of a profile image that's uploaded manually by the user via the normal process, or is there no fixed size for that either?